services/graphql-server/src/graphql/definitions/platform/content/index.js

```diff
@@ -472,6 +478,28 @@ input ContentEventDatesMutationInput {
   allDay: Boolean = false
 }
 
+"Content Id to update with UserRegistration payload"
+input ContentUserRegistrationMutationInput {
+  "Content Id"
+  id: Int!
+  "Payload to update"
+  payload: ContentUserRegistrationMutationPayloadInput!
+}
+
+"User Registration fields to update on content by id"
+input ContentUserRegistrationMutationPayloadInput {
+  "Is Registration Required"
+  isRequired: Boolean
+  "Date to start require registration"
+  startDate: Date
+  "Date to end required registration"
+  endDate: Date
+  "List of site ids to enable required registration"
+  siteIds: [ObjectID!]
+  "Which level of access is required"
+  accessLevels: [String]
```

Comment:
Ensure all array items are non-nullable and all defaults are set to empty, so:

`accessLevels: [String!] = []`
`siteIds: [ObjectID!]! = []`

Reply:
Actually, it doesn't look like the resolver is handling access levels or site IDs, so just remove these from the input definition for now